fe/fe-core/src/main/java/org/apache/doris/datasource/maxcompute/source/MaxComputeScanNode.java

```diff
@@ -363,11 +363,12 @@ private String convertLiteralToOdpsValues(OdpsType odpsType, Expr expr) throws A
                 ScalarType dstType = ScalarType.createDateV2Type();
                 return  " \"" + dateLiteral.getStringValue(dstType) + "\" ";
             }
-            case DATETIME: {
-                DateLiteral dateLiteral = (DateLiteral) literalExpr;
-                ScalarType dstType = ScalarType.createDatetimeV2Type(3);
-                return  " \"" + dateLiteral.getStringValue(dstType) + "\" ";
-            }
+            // case DATETIME: {
```

Comment:
why comment out?

Reply:
I should determine which time zone the datetime needs to be passed in.